doc/40_observatory/OBS_Augment_table_with_census.md

```diff
@@ -0,0 +1,125 @@
+
+## Name
+
+OBS_AUGMENT_TABLE_WITH_CENSUS - adds the requested dimension to the input table aggregating appropriately depending on the tables
+geometry type.
+
+## Synopsis
+
+```postgresql
+OBS_AUGMENT_TABLE_WITH_CENSUS (
+  target_table_name text,
+  column_name text
+);
+
+OBS_AUGMENT_TABLE_WITH_CENSUS (
+  target_table_name text,
+  column_name text
+);
+
+OBS_AUGMENT_TABLE_WITH_CENSUS (
+  target_table_name text,
+  column_name text,
+  time_span text DEFAULT '2009 - 2013',
+  geometry_level text DEFAULT '"us.census.tiger".block_group'
+);
+
+OBS_AUGMENT_TABLE_WITH_CENSUS (
```

Comment:
All these augmentation functions are out of scope for the May release, something we agreed upon in the tech spec and later in a meeting.

I guess this kind of misalignment is just a communication issue but we need to improve on that. I don't know how to but am fully open to suggestions.

cc/ @andrewxhill @javisantana